extensions/grpc-common/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-extensions-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-grpc-common-parent</artifactId>
+    <name>Quarkus - gRPC Common - Parent</name>
```

Comment:
Ah wait, this doesn't follow our conventions: the Parent shouldn't have a suffix but the Runtime one should:
```
[INFO] Quarkus - Flyway                                                   [pom]
[INFO] Quarkus - Flyway - Runtime                                         [jar]
[INFO] Quarkus - Flyway - Deployment                                      [jar]
```

Reply:
I copied it from the gRPC module :(
Fixed.